src/models/tools/table-links.ts

```diff
@@ -42,6 +42,15 @@ export function addLinkedTable(tableId: string) {
   }
 }
 
+export function removeLinkedTable(tableId: string) {
+  const documentId = getTableDocument(tableId);
+  if (!documentId) return;
+  const linkedTables = getLinkedTables(documentId);
+  if (!linkedTables) return;
+  const index = linkedTables.indexOf(tableId);
+  index > -1 && linkedTables.splice(index, 1);
```

Comment:
```suggestion
  (index >= 0) && linkedTables.splice(index, 1);
```
Parentheses to reduce potential ambiguity for reader. Conditional because `> -1` always reads to me as "is not equivalent to the magic sentinel value defined for nonexistent values by the `indexOf()` function" whereas `>= 0` reads as the much simpler "has a valid index".